.travis.yml

```diff
@@ -11,14 +14,23 @@ install:
 
 script:
   - "pylint --msg-template='{path}:{line}: [{msg_id}({symbol}) {obj}] {msg}' epsagon/"
-  - tox
+  - pytest
```

Comment:
explain?

Reply:
no need, we now have a separate machine for each build